src/routes/home/components/ProfessionDetails.tsx

```diff
@@ -61,17 +63,16 @@ const ProfessionDetails = ({
         />
         {discipline.leafDisciplines && (
           <NumberText>
-            {moduleAlreadyStarted && `${Math.floor(progress * discipline.leafDisciplines.length)}/`}
-            {discipline.leafDisciplines.length}
+            {doneModules}/{discipline.leafDisciplines.length}
           </NumberText>
         )}
-        <UnitText>{moduleAlreadyStarted ? labels.home.progressDescription : childrenLabel(discipline, true)}</UnitText>
+        <UnitText>{doneModules > 0 ? labels.home.progressDescription : childrenLabel(discipline, true)}</UnitText>
       </ProgressContainer>
       <NextExerciseCard
         thumbnail={documents[0].document_image[0].image}
         onPress={() => navigateToNextExercise(nextExerciseData)}
         heading={EXERCISES[exerciseKey].title}
-        buttonLabel={labels.home.continue}
+        buttonLabel={doneModules > 0 ? labels.home.continue : labels.home.start}
```

Comment:
I think here we need moduleAlreadyStarted instead of doneModules.

Reply:
Disregard my previous comment, you were right. Shows 'fortsetzen' as soon as one level (not module) has been completed now.